app/src/test/java/org/mozilla/focus/onboarding/OnBoardingStorageTest.kt

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.onboarding
+
+import androidx.preference.PreferenceManager
+import androidx.test.core.app.ApplicationProvider
+import mozilla.components.support.test.robolectric.testContext
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.doReturn
+import org.mockito.Mockito.spy
+import org.mozilla.focus.R
+import org.mozilla.focus.fragment.onboarding.OnBoardingStep
+import org.mozilla.focus.fragment.onboarding.OnBoardingStorage
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class OnBoardingStorageTest {
+    private lateinit var onBoardingStorage: OnBoardingStorage
+
+    @Before
+    fun setup() {
+        onBoardingStorage = spy(OnBoardingStorage(testContext))
+    }
+
+    @Test
+    fun `GIVEN get current onBoarding step is called WHEN pref_key_first_screen is saved in SharedPreferences THEN onBoarding step first screen is return`() {
```

Comment:
nit:
The `GIVEN - WHEN - THEN` naming refer to `preconditions - action - result`
So "GIVEN get current onBoarding step is called" would more likely be "WHEN get current onBoarding step is called"
for the entire title to maybe say
> GIVEN at the first onboarding step WHEN querying the current onboarding step from storage THEN get the first step

Looking at how this works in "OnBoardingStorage" there may also a a separate test for 
> GIVEN onboarding not started WHEN querying the current onboarding step from storage THEN get the first step

Reply:
Done